### ISSUE TITLE
limit((x**Rational(1,4)-2)/(sqrt(x)-4)**Rational(2, 3), x, 16) NotImplementedError

### ISSUE THREAD

**Issue by satels** [sympy/sympy#8229]
On master

<pre>
>>> limit((x**Rational(1,4)-2)/(sqrt(x)-4)**Rational(2, 3), x, 16)
Traceback (most recent call last): File "<string>", line 1, in <module> File "/base/data/home/apps/s~sympy-live-hrd/43.373169527249054993/sympy/sympy/series/limits.py", line 85, in limit r = gruntz(e, z, z0, dir) File "/base/data/home/apps/s~sympy-live-hrd/43.373169527249054993/sympy/sympy/series/gruntz.py", line 637, in gruntz r = limitinf(e0, z) File "/base/data/home/apps/s~sympy-live-hrd/43.373169527249054993/sympy/sympy/core/cache.py", line 93, in wrapper r = func(*args, **kw_args) File "/base/data/home/apps/s~sympy-live-hrd/43.373169527249054993/sympy/sympy/series/gruntz.py", line 416, in limitinf c0, e0 = mrv_leadterm(e, x) File "/base/data/home/apps/s~sympy-live-hrd/43.373169527249054993/sympy/sympy/core/cache.py", line 93, in wrapper r = func(*args, **kw_args) File "/base/data/home/apps/s~sympy-live-hrd/43.373169527249054993/sympy/sympy/series/gruntz.py", line 496, in mrv_leadterm series = calculate_series(f, w, logx=logw) File "/base/data/home/apps/s~sympy-live-hrd/43.373169527249054993/sympy/sympy/series/gruntz.py", line 453, in calculate_series for t in e.lseries(x, logx=logx): File "/base/data/home/apps/s~sympy-live-hrd/43.373169527249054993/sympy/sympy/core/expr.py", line 2492, in yield_lseries for si in s: File "/base/data/home/apps/s~sympy-live-hrd/43.373169527249054993/sympy/sympy/core/expr.py", line 2566, in _eval_lseries series = self._eval_nseries(x, n=n, logx=logx) File "/base/data/home/apps/s~sympy-live-hrd/43.373169527249054993/sympy/sympy/core/mul.py", line 1445, in _eval_nseries res += C.Order(x**n, x) File "/base/data/home/apps/s~sympy-live-hrd/43.373169527249054993/sympy/sympy/core/decorators.py", line 77, in __sympifyit_wrapper return func(a, b) File "/base/data/home/apps/s~sympy-live-hrd/43.373169527249054993/sympy/sympy/core/decorators.py", line 118, in binary_op_wrapper return func(self, other) File "/base/data/home/apps/s~sympy-live-hrd/43.373169527249054993/sympy/sympy/core/expr.py", line 105, in __add__ return Add(self, other) File "/base/data/home/apps/s~sympy-live-hrd/43.373169527249054993/sympy/sympy/core/cache.py", line 93, in wrapper r = func(*args, **kw_args) File "/base/data/home/apps/s~sympy-live-hrd/43.373169527249054993/sympy/sympy/core/operations.py", line 40, in __new__ c_part, nc_part, order_symbols = cls.flatten(args) File "/base/data/home/apps/s~sympy-live-hrd/43.373169527249054993/sympy/sympy/core/add.py", line 122, in flatten if o1.contains(o): File "/base/data/home/apps/s~sympy-live-hrd/43.373169527249054993/sympy/sympy/core/cache.py", line 93, in wrapper r = func(*args, **kw_args) File "/base/data/home/apps/s~sympy-live-hrd/43.373169527249054993/sympy/sympy/series/order.py", line 349, in contains l = limit(ratio, s, point) != 0 File "/base/data/home/apps/s~sympy-live-hrd/43.373169527249054993/sympy/sympy/series/limits.py", line 85, in limit r = gruntz(e, z, z0, dir) File "/base/data/home/apps/s~sympy-live-hrd/43.373169527249054993/sympy/sympy/series/gruntz.py", line 637, in gruntz r = limitinf(e0, z) File "/base/data/home/apps/s~sympy-live-hrd/43.373169527249054993/sympy/sympy/core/cache.py", line 93, in wrapper r = func(*args, **kw_args) File "/base/data/home/apps/s~sympy-live-hrd/43.373169527249054993/sympy/sympy/series/gruntz.py", line 416, in limitinf c0, e0 = mrv_leadterm(e, x) File "/base/data/home/apps/s~sympy-live-hrd/43.373169527249054993/sympy/sympy/core/cache.py", line 93, in wrapper r = func(*args, **kw_args) File "/base/data/home/apps/s~sympy-live-hrd/43.373169527249054993/sympy/sympy/series/gruntz.py", line 471, in mrv_leadterm Omega, exps = mrv(e, x) File "/base/data/home/apps/s~sympy-live-hrd/43.373169527249054993/sympy/sympy/series/gruntz.py", line 265, in mrv s, expr = mrv(b, x) File "/base/data/home/apps/s~sympy-live-hrd/43.373169527249054993/sympy/sympy/series/gruntz.py", line 300, in mrv "Don't know how to calculate the mrv of '%s'" % e) NotImplementedError: Don't know how to calculate the mrv of 'O(1/_w, (_w, oo))' 
</pre>

On 0.7.3

In [3]: limit((x**Rational(1,4)-2)/(sqrt(x)-4)**Rational(2, 3), x, 16)
Out[3]: 0

On wolfram:

http://www.wolframalpha.com/input/?i=limit+%28x^%281%2F4%29-2%29%2F%28%28sqrt%28x%29-4%29^%282%2F3%29%29+x+-%3E+16

= 0

**Comment by sachin-4099** [comment]
The issue here seems resolved, I am getting this on master:
```
In [1]: limit((x**Rational(1,4)-2)/(sqrt(x)-4)**Rational(2, 3), x, 16)
Out[1]: 0
```

**Comment by gschintgen** [comment]
Yes, no error on master. This needs a test.

**Comment by dsaw** [comment]
Have included a test for this #19922
